vowpalwabbit/parse_args.cc

```diff
@@ -1201,6 +1201,8 @@ vw& parse_args(int argc, char *argv[], trace_message_t trace_listener, void* tra
     ("initial_regressor,i", po::value< vector<string> >(), "Initial regressor(s)")
     ("initial_weight", po::value<float>(&(all.initial_weight)), "Set all weights to an initial value of arg.")
     ("random_weights", po::value<bool>(&(all.random_weights)), "make initial weights random")
+    ("normal_weights", po::value<bool>(&(all.normal_weights)), "make initial weights normal")
+    ("truncated_normal_weights", po::value<bool>(&(all.tnormal_weights)), "make initial weights truncated normal")
```

Comment:
Does truncated normal means exactly at +/- 1 SD?  What if someone wants more control?
Maybe make the interface `--normal_weights <mean>,<std>` with zero/omitted SD meaning some default.

This will need some documentation update too (in the wiki of command line args).